src/Native/JsArray.js

```diff
@@ -0,0 +1,84 @@
+// import Maybe
+
+var _elm_lang$core$Native_JsArray = function() {
+/* A thin, but still immutable, wrapper over native Javascript arrays. */
+
+// An empty array
+var empty = {
+    ctor: 'JsArray',
+    _0: []
+};
+
+function singleton(val) {
+    return {
+        ctor: 'JsArray',
+        _0: [val]
+    };
+}
+
+function length(arr) {
+    return arr._0.length;
+}
+
+function get(idx, arr) {
+    var val = arr._0[idx];
+    if (val) {
```

Comment:
This is wrong. What if the `arr` contains vals which are bools, some of which are false? I would use `typeof val === "undefined"`